Simplenote/Classes/NSSortDescriptor+Simplenote.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+
+
+// MARK: - SortMode List Methods
+//
+extension NSSortDescriptor {
+
+    /// Returns a NSSortDescriptor, to be applied over Note collections, so that the resulting collection relects the specified `SortMode`
```

Comment:
```suggestion
    /// Returns a NSSortDescriptor, to be applied over Note collections, so that the resulting collection reflects the specified `SortMode`
```

Reply:
Thank you!!